documentation/metrics_api.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: default
+title: "Puppet Server: Metrics API"
+canonical: "/puppetserver/latest/metrics_api.html"
+---
+
+# The Metrics API
+
+Puppet Enterprise includes two optional, enabled-by-default web APIs for
+[Java Management Extension (JMX)](https://docs.oracle.com/javase/tutorial/jmx/index.html)
+metrics, namely
+[managed beans (MBeans)](https://docs.oracle.com/javase/tutorial/jmx/mbeans/).
+
+## Metrics v2
```

Comment:
d'oh, I think I'm wrong about the default for Jolokia.  I think it actually is enabled by default.  Might still be good to add a boilerplate `metrics-webservice` section into the config with the `enabled` and `servlet-init-params` sections commented out, so they would be using their default values.  @justinstoller, WDYT?